### ISSUE TITLE
support: tool0 needed for URDFs

### ISSUE THREAD

**Issue by phvass** [ros-industrial/motoman#21]
Request tool0 be added to each robot, and URDFs updated.

**Comment by shaun-edwards** [comment]
Are all robots missing tool0?  If not, can you provide a list?

**Comment by phvass** [comment]
tool0 needed for the following files:

mh5_mesh_macro.xacro
sia10d_mesh_macro.xacro
sia20d_mesh_macro.xacro

**Comment by phvass** [comment]
Here is the folder with the correct xacro file from the training supplements: https://github.com/ros-industrial/industrial_training/tree/master/training/supplements/motoman_config/cfg

Could use Josiah's URDFs also. They have higher fidelity CAD detail...

**Comment by shaun-edwards** [comment]
The sia10d_mesh_macro.xacro doesn't appear to have a tool0 defined.  Is this required for the training or just the other ones?

**Comment by phvass** [comment]
Looks like only the sia20d is used in the training material. However, I think we want a tool_0 for all manipulator URDF from here out. So let's add it to the sia10d and sia5d.

Thanks!

Paul

**Comment by shaun-edwards** [comment]
Agreed.  I will port the ones that are already defined for the first Hydro release.  The others will have to wait for a later release(Hydro patch or later) since we will have to verify the geometry.

**Comment by mbharatheesha** [comment]
tool0 is still missing for the mh5/mh5L model in the Indigo release. Is there any update planned to be released soon?

**Comment by mbharatheesha** [comment]
(Gijs) just also noticed that there is no `base` link in mh5 model and sia20d model.

**Comment by gavanderhoorn** [comment]
Related: #45.
